packages/flutter/test/material/list_tile_test.dart

```diff
@@ -3616,7 +3616,7 @@ void main() {
       RenderParagraph title = _getTextRenderObject(tester, 'title');
       expect(title.text.style!.color, theme.textTheme.titleMedium!.color);
       RenderParagraph subtitle = _getTextRenderObject(tester, 'subtitle');
-      expect(subtitle.text.style!.color, theme.textTheme.bodySmall!.color);
+      expect(subtitle.text.style!.color, theme.textTheme.bodyMedium!.color);
```

Comment:
Please revert this default test update.
```suggestion
      expect(subtitle.text.style!.color, theme.textTheme.bodySmall!.color);
```